advanced/Templates/gravity.db.sql

```diff
@@ -0,0 +1,24 @@
+CREATE TABLE whitelist (domain TEXT UNIQUE NOT NULL, enabled BOOLEAN NOT NULL DEFAULT 1, date_added DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP, comment TEXT);
+CREATE TABLE blacklist (domain TEXT UNIQUE NOT NULL, enabled BOOLEAN NOT NULL DEFAULT 1, date_added DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP, comment TEXT);
+CREATE TABLE regex     (filter TEXT UNIQUE NOT NULL, enabled BOOLEAN NOT NULL DEFAULT 1, date_added DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP, comment TEXT);
+CREATE TABLE gravity   (domain TEXT UNIQUE NOT NULL);
+CREATE TABLE info      (property TEXT NOT NULL, value TEXT NOT NULL);
+
+INSERT INTO info VALUES("version","1");
+
+CREATE VIEW vw_gravity AS SELECT DISTINCT a.domain
+FROM gravity a
+WHERE a.domain NOT IN (SELECT domain from whitelist WHERE enabled == 1);
```

Comment:
This does not take into account the blacklisted domains.

Reply:
This is intended, the blacklist domains have to be loaded separately such that FTL can distinguish between gravity and user blacklisted domains. This is the same as we do it right now. `gravity.list` does not include the domains from `black.list`.